server/src/test/java/org/spine3/server/aggregate/AggregatePartRepositoryLookupShould.java

```diff
@@ -48,13 +49,18 @@
     public void setUp() {
         boundedContext = BoundedContext.newBuilder()
                                        .build();
+        final ProjectId id = ProjectId.newBuilder()
+                                      .setId(newUuid())
+                                      .build();
+        final ProjectRoot root = new ProjectRoot(boundedContext, id);
         boundedContext.register(new ProjectPartRepository(boundedContext));
         boundedContext.register(new TaskAggregateRepository(boundedContext));
     }
 
     @Test
     public void find_a_repository() {
-        AggregatePartRepositoryLookup lookup = createLookup(boundedContext, ProjectId.class, Project.class);
+        AggregatePartRepositoryLookup lookup = createLookup(boundedContext, ProjectId.class,
```

Comment:
If not one line, then each arg in a new line. Please fix it here and in the rest of occasions.